andela/learning-map-front
# WIP: #150400564 Implement Delete Comment Functionality

#### What does this PR do?
- Create UI component to aid comment deletion
- Implement logic for deleting comment
- Display user image in `CommentForm`

#### Description of Task to be completed?
When a user clicks on the delete icon of a comment which he/she created, the comment should be deleted.

#### How should this be manually tested?
- Start the application
- Log in with your `andela` email
- When you're successfully logged in, you should see the different contributions
- Click on any of the contributions
- Scroll down and comment on a `contribution`
- Once this is done, the comment should be displayed and you can click on the delete icon.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#150400564](https://www.pivotaltracker.com/story/show/150400564)

#### Screenshots (if appropriate)
![screen shot 2017-09-01 at 3 36 28 pm](https://user-images.githubusercontent.com/25608335/29974956-89eabe50-8f2c-11e7-9d70-9bc464206017.png)

#### Questions:
N/A
